scclasses/SCVim.sc

```diff
@@ -155,8 +156,24 @@ classvar <scvim_dir, <scvim_cache_dir,
         };
 
         postln("SCVim: processing help docs, this takes a little while....");
+        // regexp searching for filenames
+        glob_files = { arg folder, match;
+                var p;
+                PathName(folder).filesDo { arg x;
+                        if (match.matchRegexp(x.fileName)) {
+                                p = p.add(x.fileNameWithoutExtension);
+                                p = p.add(x.fullPath);
+                        }
+                };
+                p
+        };
+        postln("SCVim: Searching for all help files in " ++ Platform.resourceDir);
+        all_help_files =  Dictionary.newFrom(glob_files.value(Platform.resourceDir, "(?i)\\.(html|htm|rtf|schelp|scd)$"));
```

Comment:
I only fixed the missing "Help.tree" method and I added '.schelp' to the file extensions that are picked up. I was hoping that readAllStringRTF, readAllStringHTML and readAllString would handle the respective help files correctly, so we don't really need to drop '.rtf'. I'm using scvim on a headless Raspberry Pi through ssh.